conf/config.toml

```diff
@@ -20,8 +20,15 @@ initial-cluster-state = "new"
 lease = 3
 tso-save-interval = "3s"
 
+## Make sure you set the "zone" label for this PD server before enabling its Local TSO service.
+# enable-local-tso = true
+
 enable-prevote = true
 
+[labels]
+# This means **this** PD server belongs to data center "dc-1".
+zone="dc-1"
```

Comment:
Need to comment out this section?

Reply:
No need to comment out because there are no side effects I think.